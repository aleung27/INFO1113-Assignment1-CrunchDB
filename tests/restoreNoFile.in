RESTORE restoreNoFile.txt
LIST ENTRIES
LIST KEYS
LIST SNAPSHOTS
BYE
