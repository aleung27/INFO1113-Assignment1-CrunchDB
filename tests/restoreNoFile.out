> file not found

> no entries

> no keys

> no snapshots

> bye
